<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inscrição</string>
  </property>
  <widget class="QLabel" name="labelAluno">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Aluno:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboAlunos">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelProfessor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Professor:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboProfessores">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelSala">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sala:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboSalas">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDisciplina">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Disciplina:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboDisciplinas">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalvar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>